packages/react/src/components/SelectUsersModal/_select-users-modal.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--select-users-modal {
+  .bx--modal-container .bx--modal-content {
+    padding-right: $spacing-05;
+  }
+
+  .#{$iot-prefix}--list {
+    background-color: #f4f4f4;
+    height: 100%;
+  }
+  .#{$prefix}--search-input {
+    background-color: #fff;
```

Comment:
should be using carbon color tokens with the gray10 theme. Can find tokens [here](https://www.carbondesignsystem.com/guidelines/color/usage)